CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 * **BUGFIX** (by @imaNNeo) Fix Negative BarChartRodStackItem are not drawn correctly bug, #1347
 * **BUGFIX** (by @imaNNeo) Fix bar_chart_helper minY calculation bug, #1388
 * **IMPROVEMENT** (by @imaNNeo) Consider fraction digits when formatting chart side titles, #1267
+* **IMPROVEMENT** (by @devilk10) Use the default theme available by the themeData for tooltips instead of color params #1464
```

Comment:
You need to add this line in the new version tag, for example create a `## newVersion` section and add your changelog there
(Please read the [contributing guideline](https://github.com/imaNNeo/fl_chart#contributing) carefully.